Add link to c/common contributor's guide (#2280)
containers/storage: CONTRIBUTING.md

# Contributing to Containers/Storage

We'd love to have you join the community! [Learn how to contribute](https://github.com/containers/common/blob/main/CONTRIBUTING.md) to the Containers Group Projects.

> Please note that this information cannot be relevant to this project:

Reviewer: ```suggestion
Please note that the following information is specific to this project:
```

> - We don’t typically require 2 LGTMs — few people are watching the repo

Reviewer: ```suggestion
 * We don’t typically require 2 LGTMs for this repository.
```
